Sources/StreamVideoSwiftUI/Utils/PictureInPicture/StreamPictureInPictureVideoRenderer.swift

```diff
@@ -113,45 +113,49 @@ final class StreamPictureInPictureVideoRenderer: UIView, RTCVideoRenderer {
     // MARK: - Rendering lifecycle
 
     /// This method is being called from WebRTC and asks the container to set its size to the track's size.
-    func setSize(_ size: CGSize) {
-        trackSize = size
+    nonisolated func setSize(_ size: CGSize) {
+        DispatchQueue.main.async {
```

Comment:
Can we prefer task @mainactor please? I don't see a point on doing migration to full concurrency support and use dispatch queue main

Reply:
yes, we should - old habits 😁